### PR TITLE
feat: Added readme

### PR DESCRIPTION
**Pull request by SolMendiola**
# :star: Feature

---

#### :pencil2: Description:

Added readme

---

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,60 +1,34 @@
...
+# Fluttips
 
-## Arch Overview 
+## Introduction
 
-The project is divided into two main folders:
-- The UI contains all app screens.
-- The Core contains the models and the data layer.
+Fluttips is the first application that provides tips for programming in Flutter.
+It provides different sections, for example, you can see images with code explications and if you like, you can save the tip to see it later in favorites section.
```
Comment (ValentinaBianco) [inline README.md:6]:
```suggestion
Some of its features include images and code explanations, making it easy to understand the tips provided.
Additionally, users can save their favorite tips to a "favorites" section, allowing them to easily access them later.
```

File: README.md [hunk 2]
```diff
@@ -1,60 +1,34 @@
...
+In a coming soon version, you will see videos of the widgets of the week, that provides you information about different components of Flutter.
```
Comment (ValentinaBianco) [inline README.md:7]:
```suggestion
In an upcoming version, you will be able to view "Widget of the Week" videos, which will provide you with detailed information about various elements of Flutter.
```

File: README.md [hunk 3]
```diff
@@ -1,60 +1,34 @@
...
+  </tr>
+  <tr>
+    <th>Favorites</th>
+    <th><img src="readme/Favorites.gif" width="250"/></th>
+  </tr>
+</table>
 
-[Flutter Bloc][bloc] is used for state management, specifically, we use Cubit to manage the screen state.
-Each app section is added in a project folder which contains three components, the Screen (a `StatelessWidget`, the UI), the Cubit and the state.
+The application was made with love by [Xmartlabs][xmartlabs] team and Fluttips is based on [Xmartlabs template][xmartlabs_template], which provides a clean architecture.
```
Comment (ValentinaBianco) [inline README.md:20]:
```suggestion
Made with ❤️ by [Xmartlabs][xmartlabs] and based on [our template][xmartlabs_template], which provides a clean architecture.
```

File: README.md [hunk 4]
```diff
@@ -1,60 +1,34 @@
...
+  <tr>
+    <th>Favorites</th>
+    <th><img src="readme/Favorites.gif" width="250"/></th>
+  </tr>
+</table>
 
-[Flutter Bloc][bloc] is used for state management, specifically, we use Cubit to manage the screen state.
-Each app section is added in a project folder which contains three components, the Screen (a `StatelessWidget`, the UI), the Cubit and the state.
+The application was made with love by [Xmartlabs][xmartlabs] team and Fluttips is based on [Xmartlabs template][xmartlabs_template], which provides a clean architecture.   
+The images tips are from [Vandadnp][Vandadnp]
```
Comment (ValentinaBianco) [inline README.md:21]:
```suggestion
👉 The image tips are from [Vandadnp][Vandadnp]
```

File: README.md [hunk 5]
```diff
@@ -1,60 +1,34 @@
...
+and the videos are from [Widgets of the week][Widget_of_the_week].
 
-The `MainScreen` is the Widget that contains all screens. It defines the `MaterialApp` and provides the app router.
-The router has two subgraphs, the `UnauthenticatedRouter` used for unauthenticated users and the `AuthenticatedRouter` used for authenticated users.
+## Contribute
 
-The [app router][app_router] is provided by [auto_route][auto_route], and contains the previous sections with some nested screens.
+- If you **want to contribute** please feel free to **submit pull requests**.
+- If you **have a feature request** please **open an issue**.
+- If you **found a bug** or **need help** please let us know.
```
Comment (ValentinaBianco) [inline README.md:29]:
```suggestion
👉 If you enjoy using **Fluttips** we would love to hear about it! Drop us a line on [twitter](https://twitter.com/xmartlabs).
```

**Review by ValentinaBianco — CHANGES_REQUESTED** [review]
Readme changes

File: README.md [hunk 6]
```diff
@@ -1,60 +1,36 @@
...
+  </tr>
+  <tr>
+    <th>Favorites</th>
+    <th><img src="readme/Favorites.gif" width="250"/></th>
+  </tr>
+</table>
+
+Made with ❤️ by [Xmartlabs][xmartlabs] and based on [our template][xmartlabs_template], which provides a clean architecture.
+👉 The image tips are from [Vandadnp][Vandadnp]
+👉 The videos are from [Widgets of the week][Widget_of_the_week].
```
Comment (mirland) [inline README.md:24]: check the preview but new lines are missing

File: README.md [hunk 7]
```diff
@@ -1,60 +1,36 @@
...
+
+## Contribute
+
+  👉 If you **want to contribute** please feel free to **submit pull requests**.
+  👉 If you **have a feature request** please **open an issue**.
```
Comment (mirland) [inline README.md:29]: link to new issue

File: README.md [hunk 8]
```diff
@@ -1,60 +1,36 @@
...
+  👉 If you **found a bug** or **need help** please let us know.
+  👉 If you enjoy using **Fluttips** we would love to hear about it! Drop us a line on [twitter](https://twitter.com/xmartlabs).
```
Comment (mirland) [inline README.md:31]: ref?

File: README.md [hunk 9]
```diff
@@ -4,33 +4,37 @@
...
-  </tr>
-  <tr>
-    <th>Favorites</th>
-    <th><img src="readme/Favorites.gif" width="250"/></th>
-  </tr>
-</table>
+
+<img src="readme/Images.gif" width="250"/>
+
+<img src="readme/Favorites.gif" width="250"/>
```
Comment (mirland) [inline README.md:16]: what about adding both in the same row?

File: README.md [hunk 10]
```diff
@@ -1,60 +1,40 @@
-# Xmartlabs' Flutter template
+# Fluttips
```
Comment (mirland) [inline README.md:1]: We should add a banner with the logo

File: README.md [hunk 11]
```diff
@@ -1,60 +1,40 @@
...
```
Comment (mirland) [inline README.md:2]: include badges

File: README.md [hunk 12]
```diff
@@ -1,60 +1,40 @@
...
 
-The `MainScreen` is the Widget that contains all screens. It defines the `MaterialApp` and provides the app router.
-The router has two subgraphs, the `UnauthenticatedRouter` used for unauthenticated users and the `AuthenticatedRouter` used for authenticated users.
 
-The [app router][app_router] is provided by [auto_route][auto_route], and contains the previous sections with some nested screens.
+<img src="readme/Images.gif" width="250"/>
 
-The [theme folder][theme] contains the theme definitions, for example colors, styles and dimensions. 
-Another important plugin related to the UI is [flutter_screenutil][flutter_screenutil], used for adapting screen and font size.
+<img src="readme/Favorites.gif" width="250"/>
```
Comment (mirland) [inline README.md:16]: what about adding both in the same row?

File: README.md [hunk 13]
```diff
@@ -1,60 +1,40 @@
...
-You can do it by executing the following command:
+  👉 If you **want to contribute** please feel free to **submit pull requests**.
 
-```sh
-./scripts/setup_firebase.sh path_to_config_folder
-```
+  👉 If you **have a feature request** please **[open an issue][new_issue]**.
 
-The config folder should contain the `google-services.json`, `GoogleService-Info-prod.plist` and `GoogleService-Info-dev.plist` files.
+  👉 If you **found a bug** or **need help** please let us know.
```
Comment (mirland) [inline README.md:31]: how? maybe a link

**Review by ValentinaBianco — DISMISSED** [review]
small changes to readme

File: README.md [hunk 14]
```diff
@@ -1,4 +1,4 @@
-# Fluttips
+<img src="readme/Fluttips_logo.png" width=“100%”/>
```
Comment (mirland) [inline README.md:1]: the image does not fit in the screen
image

File: README.md [hunk 15]
```diff
@@ -1,60 +1,38 @@
-# Xmartlabs' Flutter template
+<img src="readme/Fluttips_logo.png" width=“100%”/>
 
-## Arch Overview 
+[![Flutter CI](https://github.com/xmartlabs/fluttips/actions/workflows/flutter-ci.yml/badge.svg)](https://github.com/xmartlabs/fluttips/actions/workflows/flutter-ci.yml)
 
-The project is divided into two main folders:
-- The UI contains all app screens.
-- The Core contains the models and the data layer.
+## Introduction
```
Comment (mirland) [inline README.md:5]: Do we need this title?

File: README.md [hunk 16]
```diff
@@ -1,60 +1,38 @@
...
-Each app section is added in a project folder which contains three components, the Screen (a `StatelessWidget`, the UI), the Cubit and the state.
+Some of its features include images and code explanations, making it easy to understand the tips provided.
+Additionally, users can save their favorite tips to a "favorites" section, allowing them to easily access them later.
 
-The `MainScreen` is the Widget that contains all screens. It defines the `MaterialApp` and provides the app router.
-The router has two subgraphs, the `UnauthenticatedRouter` used for unauthenticated users and the `AuthenticatedRouter` used for authenticated users.
+In an upcoming version, you will be able to view "Widget of the Week" videos, which will provide you with detailed information about various elements of Flutter.
 
-The [app router][app_router] is provided by [auto_route][auto_route], and contains the previous sections with some nested screens.
+<img src="readme/Images.gif" width=“500”/> <img src="readme/Favorites.gif" width="500"/>
```
Comment (mirland) [inline README.md:15]: the image does not look well 
image

File: README.md [hunk 17]
```diff
@@ -1,60 +1,38 @@
...
 
-## Firebase Setup
-To run this project, you need to config the Firebase project. 
-You can do it by executing the following command:
+  👉 If you **found a bug** or **[need help][new_issue]** please let us know.
 
-```sh
-./scripts/setup_firebase.sh path_to_config_folder
-```
+  👉 If you enjoy using **Fluttips** we would love to hear about it! Drop us a line on [twitter][twitter].
```
Comment (mirland) [inline README.md:32]: license should be added here

File: README.md (hunk 16 above)
Comment (mirland) [inline README.md:15]: images shouldn't say debug

File: README.md [hunk 18]
```diff
@@ -1,60 +1,38 @@
-# Xmartlabs' Flutter template
+<img src="readme/Fluttips_logo.png" width=“100%”/>
```
Comment (mirland) [inline README.md:1]: I'm not sure about the folder, what about /docs/assets?

**Review by mirland — CHANGES_REQUESTED** [review]

**Pull request merged**